depends/README.md

```diff
@@ -53,6 +53,18 @@ To build dependencies for the current arch+OS:
 
     gmake
 
+### Alpine
+
+    apk add bash build-base cmake curl make patch
+
+Skip the following packages if you don't intend to use the GUI and will build with [`NO_QT=1`](#dependency-options):
+
+    apk add bison linux-headers ninja-build pkgconf python3 xz
```

Comment:
1. I think we should consider a note from `apk`:
```
Executing ninja-build-1.12.1-r1.post-install
* this only installs ninja to /usr/lib/ninja-build/bin/ninja
* add that to your path to use it, or invoke it directly.
* for most uses, you want samurai instead:
*  $ apk add samurai
* which has a "ninja" executable compatible with ninja.
```
and install `samurai`. Otherwise, Qt still complains:
```
CMake Warning at qtbase/cmake/QtBuildHelpers.cmake:12 (message):
  The officially supported CMake generator for building Qt is Ninja / Ninja
  Multi-Config.  You are using: 'Unix Makefiles' instead.  Thus, you might
  encounter issues.  Use at your own risk.
```

2. Is `xz` really necessary?

Reply:
Updated to `samurai` and removed explicit `xz` (seems preferable to let this be pulled in as a dependency, or not, by `cmake`) in 4f502baf8f649e30d9495760b54080c272882213

Reply:
> seems preferable to let this be pulled in as a dependency, or not, by cmake

Not sure? It's a dep for Qt, and some of its sub deps. Nothing to do with CMake.